### ISSUE TITLE
Ability to change the background color of (e.g timeseries) panels

### ISSUE THREAD

**Issue by AntoineThebaud** [perses/perses#1541]
### Proposal

Proposal from my fellow amadeus colleague @ahurtaud: It could be useful to be able to change the background color of a panel. E.g you have a timeseries panel dedicated to error trends, showing multiple of them. In such case instead of putting the series in red, which could make the graph hard to read, putting the background in red & letting the series color in auto mode would allow for easier understanding of the panel & a better highlight of it in the dashboard.

NB: probably the color choice should not be fully free here but instead a restricted list of choice, that would take into account the currently-active theme.

**Comment by ahurtaud** [comment]
Thanks, to complete a bit more about my usecase.
I usually build dashboards with a left column of "success" and a right column about the same metrics on "failures/error". Having a background colour transparent-red would help viewers of my dashboards.

This option exists in GF for tables, singlestat, but not timeseries. And I am talking about changing this grey area of panel):
![Screenshot 2023-10-30 at 09 34 26](https://github.com/perses/perses/assets/25080995/3561655a-0240-40bd-bfec-e5b44d5198f7)
![Screenshot 2023-10-30 at 09 35 49](https://github.com/perses/perses/assets/25080995/7302c2a5-b693-4c9c-8e18-09f752aead75)

**Comment by sjcobb** [comment]
Would using our StatChart with the sparkline enabled be an option? That automatically sets the color based on thresholds currently, but if something is missing in that panel type, it could be easier to adjust to meet this use case

**Comment by AntoineThebaud** [comment]
Sorry @sjcobb kinda forgot to reply here.. Well the idea was mainly for the Timeseries panel here, the Statchart doesn't fit when you want to display multiple series on the same panel and/or when you want precise info on Y axis. But we could think of other panel types and/or enabling this for all panel types, since it would be quite de-correlated from the content. E.g it could come in handy to make a warning section using a markdown panel with orange background.

Basically in my mind it would look a bit like this:
![customBgColor](https://github.com/perses/perses/assets/7058693/ceb4031d-01a3-4566-bbe9-07faa66154f4)
